### PR TITLE
Pr4-4

### PR DESCRIPTION
**Pull request by SandaPanteleimonov**
Following merge includes addition of new object and array.
Functions were removed.
Function definitions modified.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,36 @@
...
-        - № (just an index number from 1 to 5)
-        - Name
-        - Picture (store pictures locally, - create a folder called ‘assets’)
-        - Description (a short description of the flower)
-    3. Make a ‘zebra’ view of the table
-    4. Add a CSS style that changes background color of a table row’s on hover
-    5. Use palette when picking colors
-2. Create pull request, where source branch is ‘<username>-hw3’, and target branch is ‘master’ 
-3. Follow the reviewer's suggestions (if any) until the branch is merged
+<!DOCTYPE html>
```
Comment (eclimov) [inline README.md]: Updates in this file were not required by the task.
Looks like you have accidentally committed it.
Please, rollback the changes.

File: public/hw2-united.html [hunk 2]
```diff
@@ -169,7 +169,7 @@ <h1>Buttons</h1>
 
         <div id="section5">
             <h1>Palette</h1>
-            <img title="colorpalette" src="./sanda/palette.png" alt="You have Internet Explorer, sorry" id="palette">
+            <img title="colorpalette" src="sanda/practice/palette.png" alt="You have Internet Explorer, sorry" id="palette">
```
Comment (eclimov) [inline public/hw2-united.html]: Updates in this file were not required by the task.

File: public/index.html [hunk 3]
```diff
@@ -169,7 +169,7 @@ <h1>Buttons</h1>
 
     <div id="section5">
       <h1>Palette</h1>
-      <img title="colorpalette" src="./sanda/palette.png" alt="You have Internet Explorer, sorry" id="palette">
+      <img title="colorpalette" src="sanda/practice/palette.png" alt="You have Internet Explorer, sorry" id="palette">
```
Comment (eclimov) [inline public/index.html]: Updates in this file were not required by the task.

File: public/sanda/practice/hw3.html [hunk 4]
```diff
@@ -49,31 +49,31 @@ <h2>Basic HTML Table</h2>
     <tr>
         <td>1</td>
         <td>Castor oil plant “Palm of Christ”</td>
-        <td><img src="assets/castor.jfif" alt="Castor"></td>
+        <td><img src="../assets/castor.jfif" alt="Castor"></td>
```
Comment (eclimov) [inline public/sanda/practice/hw3.html]: Updates in this file were not required by the task.

**Review by eclimov — CHANGES_REQUESTED** [review]
Please, revert unnecessary changes.

**Review by eclimov — APPROVED** [review]
Good job. Approved.

**Review by tzatushevskaya — APPROVED** [review]

**Pull request merged**